code/game/jobs/access.dm

```diff
@@ -99,7 +98,10 @@
 		ACCESS_CIVILIAN_COMMAND,
 	)
 
-/proc/get_all_marine_access()
+/proc/get_all_marine_accesses()//Includes restricted accesses
+	return list(ACCESS_MARINE_CAPTAIN) + get_main_marine_accesses()
```

Comment:
Would be better if this was `get_all_marine_access` so it conformed to the naming scheme of the rest of the procs.

Also, code-doc:
```suggestion
///Includes restricted accesses
/proc/get_all_marine_accesses()
	return list(ACCESS_MARINE_CAPTAIN) + get_main_marine_accesses()
```